libqtile/layout/xmonad.py

```diff
@@ -138,9 +138,9 @@ class MonadTall(Layout):
 
     """
 
-    _left = 0
-    _right = 1
-    _min_height = 85
+    _align_min = 0
+    _align_max = 1
```

Comment:
yep, I totally agree. I was trying to find a generic name for this, but clearly without success :-) will do that.